### PR TITLE
Add http/server/config to configure http timeouts

### PR DESCRIPTION
**Pull request by Danielius1922**

### CODE REVIEW

**Comment by jkralik** [comment]
How this timeout's affects websockets?

File: http-gateway/config.yaml [hunk 1]
```diff
@@ -9,6 +9,10 @@ log:
 apis:
   http:
     address: "0.0.0.0:9100"
+    readTimeout: 10s
+    readHeaderTimeout: 10s
```
Comment (jkralik) [inline http-gateway/config.yaml]: Here is good description how these timeout works:
https://blog.cloudflare.com/the-complete-guide-to-golang-net-http-timeouts/

So i suggest to change default behavior to this values:
```
readHeaderTimeout: 4s
readTimeout: 8s
writeTimeout: 16s
idleTimeout: 30s
```

**Review by jkralik — APPROVED** [review]
Pls update doc configs at: https://github.com/plgd-dev/doc/tree/main/content/en/configuration

**Comment by Danielius1922** [comment]
> Pls update doc configs at: https://github.com/plgd-dev/doc/tree/main/content/en/configuration

https://github.com/plgd-dev/doc/pull/66

**Pull request merged**